src/pages/ListView.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import db from '../lib/firebase';
+import { collection } from 'firebase/firestore';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
+export const ListView = ({ token }) => {
+  const [value, loading, error] = useCollection(collection(db, token), {
```

Comment:
You might want to check if the `token` exists before contacting Firebase. I think that's the source of the issue Yenly noticed.